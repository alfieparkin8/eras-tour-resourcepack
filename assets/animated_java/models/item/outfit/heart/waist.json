{"parent":"animated_java:item/outfit/waist","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}