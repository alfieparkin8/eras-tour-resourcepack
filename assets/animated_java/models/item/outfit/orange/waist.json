{"parent":"animated_java:item/outfit/waist","textures":{"2":"minecraft:block/outfits/1989/orange_layer_1","4":"minecraft:block/outfits/1989/orange_layer_1","6":"minecraft:block/outfits/1989/orange_layer_2"}}