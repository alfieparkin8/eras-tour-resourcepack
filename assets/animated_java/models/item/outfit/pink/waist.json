{
	"parent": "animated_java:item/outfit/waist",
	"textures": {
		"2": "animated_java:item/outfit/pink_layer_1",
		"4": "animated_java:item/outfit/pink_layer_1",
		"6": "animated_java:item/outfit/pink_layer_2"
	}
}