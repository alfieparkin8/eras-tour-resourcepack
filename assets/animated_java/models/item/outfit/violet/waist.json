{
	"parent": "animated_java:item/outfit/waist",
	"textures": {
		"0": "minecraft:block/outfits/lover/golden_layer_1",
		"1": "minecraft:block/outfits/lover/heart_layer_1",
		"2": "minecraft:block/outfits/lover/violet_layer_1",
		"3": "minecraft:block/outfits/lover/violet_layer_1",
		"4": "minecraft:block/outfits/lover/violet_boots_layer_1",
		"5": "minecraft:block/outfits/lover/golden_boots_layer_1",
		"6": "animated_java:item/outfit/blank",
		"7": "animated_java:item/outfit/blue_layer_1",
		"8": "animated_java:item/outfit/blue_layer_2",
		"9": "animated_java:item/outfit/green_layer_1",
		"10": "animated_java:item/outfit/green_layer_2",
		"11": "animated_java:item/outfit/orange_layer_1",
		"12": "animated_java:item/outfit/orange_layer_2",
		"13": "animated_java:item/outfit/pink_layer_1",
		"14": "animated_java:item/outfit/pink_layer_2",
		"15": "minecraft:block/outfits/lover/tangerine_boots_layer_1",
		"16": "minecraft:block/outfits/lover/tangerine_layer_1",
		"17": "minecraft:block/outfits/lover/heart_boots_layer_1",
		"18": "minecraft:block/outfits/lover/violet_boots_layer_1"
	}
}