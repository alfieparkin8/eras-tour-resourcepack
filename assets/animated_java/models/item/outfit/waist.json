{
	"textures": {
		"6": "animated_java:item/outfit/blank"
	},
	"elements": [
		{
			"from": [
				3.9245283018867925,
				-16,
				6.6415094339622645
			],
			"to": [
				12.075471698113208,
				-4.2264150943396235,
				11.169811320754718
			],
			"rotation": {
				"angle": 0,
				"axis": "y",
				"origin": [
					8,
					-13.735849056603776,
					8
				]
			},
			"faces": {
				"north": {
					"uv": [
						5,
						10,
						7,
						16
					],
					"texture": "#6"
				},
				"east": {
					"uv": [
						7,
						10,
						8,
						16
					],
					"texture": "#6"
				},
				"south": {
					"uv": [
						8,
						10,
						10,
						16
					],
					"texture": "#6"
				},
				"west": {
					"uv": [
						4,
						10,
						5,
						16
					],
					"texture": "#6"
				},
				"up": {
					"uv": [
						0,
						0,
						0,
						0
					],
					"texture": "#6"
				},
				"down": {
					"uv": [
						0,
						0,
						0,
						0
					],
					"texture": "#6"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}