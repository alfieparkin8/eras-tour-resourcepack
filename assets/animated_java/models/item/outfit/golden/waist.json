{"parent":"animated_java:item/outfit/waist","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1","4":"minecraft:block/outfits/lover/golden_boots_layer_1"}}